assets/js/blocks/cart-checkout/cart/edit.js

```diff
@@ -37,14 +90,20 @@ const CartEditor = ( { className } ) => {
 				] }
 				defaultView={ 'full' }
 				render={ ( currentView ) => (
-					<Fragment>
+					<>
```

Comment:
I don't have any preference, but we are always explicitly writing `Fragment` in tags. Should we keep it consistent? We could even add an [eslint rule](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md) for it.

Reply:
I have a preference for fewer keystrokes and fewer imports, but If we're always using `Fragment` then I can switch back to it.

Reply:
Yeah, makes sense... We could open an issue and discuss it there, I just feel that keeping it consistent makes code easier to scan, whichever way we go.

Reply:
I know I'm contradicting myself :slightly_smiling_face:  but in [another PR](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/1736#discussion_r380131662) we were discussing if the short syntax (`<>`) would be better.

I created an issue to discuss/switch: #1761, this way we don't block current open PRs because of that.